### PR TITLE
fix the incorrect dockerimages in the ReadMe

### PR DESCRIPTION
**Pull request by aaaallleen**
Linked the correct rebuilt images 0.0.1-torch2.3.0-ubuntu22.04-cuda12.1 to the readme files.
The documentation page also needs to be modified

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,14 +53,16 @@ To facilitate setup and reproducibility, we provide Docker images via the GitHub
...
 To pull the Docker image:
 ```bash
-docker pull ghcr.io/eole-nlp/eole:0.0.1-ubuntu22.04-cuda12.1
+docker pull ghcr.io/eole-nlp/0.0.1-torch2.3.0-ubuntu22.04-cuda12.1 
 ```
 
 Example one-liner to run a container and open a bash shell within it:
 ```bash
-docker run --rm -it --runtime=nvidia ghcr.io/eole-nlp/eole:0.0.1-ubuntu22.04-cuda12.1
+docker run --rm -it --runtime=nvidia ghcr.io/eole-nlp/0.0.1-torch2.3.0-ubuntu22.04-cuda12.1
```
Comment (francoishernandez) [inline README.md:65]:
```suggestion
docker run --rm -it --runtime=nvidia ghcr.io/eole-nlp/eole:0.0.1-torch2.3.0-ubuntu22.04-cuda12.1
```

**Review by francoishernandez — CHANGES_REQUESTED** [review]
Typo. Also, you can remove the obsolete[`onmt.train.fp16.transformer.yaml`](https://github.com/eole-nlp/eole/blob/main/recipes/wmt17/onmt.train.fp16.transformer.yaml) while you're at it.
Thanks!

File: README.md (hunk 1 above)
Comment (aaaallleen) [reply README.md:65]: Sure!

Comment (aaaallleen) [reply README.md]: I think it's done!

**Review by francoishernandez — APPROVED** [review]

**Pull request merged**